pyaedt/misc/aedtlib_personalib_install.py

```diff
@@ -81,8 +81,11 @@ def install_toolkit(toolkit_dir, product):
     os.makedirs(lib_dir, exist_ok=True)
     os.makedirs(tool_dir, exist_ok=True)
     files_to_copy = ["Console", "Run_PyAEDT_Script", "Jupyter"]
-    jupyter_executable = sys.executable.replace("python.exe", "jupyter.exe")
-    ipython_executable = sys.executable.replace("python.exe", "ipython.exe")
+    # Remove hardcoded version number from python virtual environment path, so it runs version specific virtual
```

Comment:
```suggestion
    # Remove hard-coded version number from Python virtual environment path, so it runs version specific virtual
```

Reply:
@MaxJPRey I find the ending of the comment in line 84 unclear. Can it be changed to something like this:
# Remove hard-coded version number from Python virtual environment path so it runs the version specified virtually

Reply:
@PipKat  replaced with 
```py
# Remove hard-coded version number from Python virtual environment path, and replace it with the corresponding AEDT
# version's Python virtual environment.
```